src/platform_impl/linux/window.rs

```diff
@@ -647,6 +669,20 @@ impl Window {
       log::warn!("Fail to send skip taskbar request: {}", e);
     }
   }
+
+  pub fn theme(&self) -> Theme {
+    if let Some(settings) = Settings::default() {
+      let theme_name = settings.gtk_theme_name().map(|s| s.as_str().to_owned());
+      if let Some(theme) = theme_name {
+        // Currently GTK doesn't provide feature for detect theme, so we need to check theme manually.
+        // ref: https://github.com/WebKit/WebKit/blob/e44ffaa0d999a9807f76f1805943eea204cfdfbc/Source/WebKit/UIProcess/API/gtk/PageClientImpl.cpp#L587
+        if theme.ends_with("-dark") || theme.ends_with("-Dark") {
```

Comment:
I use Arc-Darker theme :( really sucks having to detect the theme this way @keiya01 @wusyong

Reply:
probably should be`
```suggestion
        if theme.contains("-dark") || theme.contains("-Dark") {
```

Reply:
Darker ... I see. I will fix this issue.

Reply:
@lucasfernog 
I created the PR. Could you check it when you have time?
https://github.com/tauri-apps/tao/pull/511